### ISSUE TITLE
Import Error in pyi_rth_qt5plugins.py on Windows Clients

### ISSUE THREAD

**Issue by greenorca** [pyinstaller/pyinstaller#4175]
I experienced an issue app running well on some, but not all Win clients with a PySide2 app build with the following packages:

- pyinstaller==3.14 
- PySide2==5.12.2

```
Traceback (most recent call last):
  File "site-packages\PyInstaller\loader\rthooks\pyi_rth_qt5plugins.py", line 47, in <module>
ImportError: DLL load failed:
```
_The Problem was solved (on some clients) by installing the Visual C++2015 Redistribution Package (x86) on the Windows 10 clients: https://www.microsoft.com/en-us/download/details.aspx?id=53840

Yet I consider this issue still work in progress...

Such requirements might perhaps be written somewhere in the *pyinstaller* dokumentation?
This might be related to: https://github.com/pyinstaller/pyinstaller/issues/2455

**Comment by bjones1** [comment]
This was fixed in 5b5791285332003940f4bc70c32bb0cf10a32917. Please reopen if there are still problems.

**Comment by greenorca** [comment]
Problem might have been be caused by running pyinstaller on virtual Win10 (Virtualbox client) and later executing the compiled exe on a native Win10 machine...
